src/utils.cpp

```diff
@@ -1516,13 +1516,14 @@ int get_skill(struct char_data *ch, int skill, int &target)
   // Iterate through their bioware, looking for anything important.
   if (should_gain_physical_boosts && ch->bioware) {
     for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content) {
-      if (GET_BIOWARE_TYPE(bio) == BIO_REFLEXRECORDER && GET_OBJ_VAL(bio, 3) == skill) {
-        strlcat(gskbuf, "Reflex Recorder skill increase: 1. ", sizeof(gskbuf));
-        skill_dice++;
-      } else if (GET_BIOWARE_TYPE(bio) == BIO_ENHANCEDARTIC)
+      if (GET_OBJ_VAL(bio, 0) == BIO_REFLEXRECORDER) {
+        recorder = GET_OBJ_VAL(bio, 5);
+      } else if (GET_OBJ_VAL(bio, 0) == BIO_ENHANCEDARTIC)
         enhan = TRUE;
-      else if (GET_BIOWARE_TYPE(bio) == BIO_SYNTHACARDIUM)
-        synth = GET_BIOWARE_RATING(bio);
+      else if (GET_OBJ_VAL(bio, 0) == BIO_SYNTHACARDIUM)
+        synth = GET_OBJ_VAL(bio, 1);
```

Comment:
`synth = GET_BIOWARE_RATING(bio);`

Reply:
Actually, you should just restore the code that's being deleted here, you want the bioware type and bioware rating macros intact.